lib/libmuser.c

```diff
@@ -45,6 +45,7 @@
 #include <stddef.h>
 #include <sys/mman.h>
 #include <stdarg.h>
+#include <linux/vfio.h>
```

Comment:
Why is this needed?

Reply:
Actually that slipped here, should probably have been a separate commit, but I don't think it's worth separating because only Travis breaks. Travis couldn't build libmuser without it, presumably because `libmuser.c` uses VFIO pound-defined constants that happen to be available via some other include or source file in the same compilation unit. I didn't spend absolutely any time thinking about why it didn't work, I just added it here.